os/arch/Kconfig

```diff
@@ -773,9 +799,18 @@ config RAM_VSTART
 		refers to the RAM that you link program code into.  If program code
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
-
+if 0
+config RAM_SIZE
+        int "Primary RAM size"
+        default 0
+        ---help---
+                The size in bytes of the installed primary RAM. "Primary" RAM
+                refers to the RAM that you link program code into.  If program code
+                does not execute out of RAM but from FLASH, then you may designate
+                any block of RAM as "primary."
+endif
 config RAM_KREGIONx_START
-	string "An address or list of start address for kernel RAM region"
+	hex "An address or list of start address for kernel RAM region"
```

Comment:
Why are you changing this to hex? Its actually a string in the defconfig. Later it will be converted into hex value in the kregion array.